helm-charts/FATE-Exchange/templates/nginx.yaml

```diff
@@ -178,6 +178,9 @@ spec:
       {{- end }}
       protocol: TCP
   type: {{ .Values.modules.nginx.type }}
+  {{- if .Values.modules.nginx.loadBalancerIP }}
+  loadBalancerIP: "{{ .Values.modules.nginx.loadBalancerIP }}"
```

Comment:
better to define a "loadBalancerIP" template

Reply:
I think it is not better to define a template, because each module has a different definition.